tests/ui-toml/toml_disallowed_methods/conf_disallowed_methods.rs

```diff
@@ -1,5 +1,3 @@
-//@compile-flags: --crate-name conf_disallowed_methods
```

Comment:
Is there any reason for this being here? I checked others in `ui-toml` and quite a few have this. I vaguely remember about this, but I don't have the specifics...

Reply:
> Is there any reason for this being here?

I assumed it was some kind of a leftover, since `conf_disallowed_methods` is the name that `ui_test` chooses for the test crate. (Aside: with the `compile-flags` in place, `ui_test` cannot test the fix.)

https://github.com/rust-lang/rust-clippy/commit/1e1ac2b4985a1236003ef23b92e3edf3af8f5dfe appears to be the commit that introduced the original directive. @Alexendoo do you recall why that was?

Reply:
My guess would be that it didn't get the right crate name at the time for whatever reason, we would've been on an (old) `compiletest-rs` version back then rather than `ui_test`